webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/detail/AnnotationDetailEditorPanel.java

```diff
@@ -479,7 +479,8 @@ public void actionFillSlot(AjaxRequestTarget aTarget, CAS aCas, int aSlotFillerB
 
         AnnotatorState state = getModelObject();
         
-        if (isUserViewingOthersWork()) {
+        if (AnnotationPageBase.isUserViewingOthersWork(getModelObject(),
```

Comment:
You could consider moving `isViewingOtherUsersWork` into `AnnotatorState` with only the current user as the argument. I think that would be nice than having it as a static method.